README.md

```diff
@@ -23,6 +23,8 @@ Code released under the [MIT License](https://opensource.org/licenses/MIT).
 
 These instructions below should work for Ubuntu 16.04. Probably it will work on other OSes as well with minor modifications.
 
+For Windows: **Before** cloning the repository use `git config --global core.autocrlf false` to enable Linux style line ending. This is necessary for the server to work. Otherwise the error `/usr/bin/env: ‘bash\r’: No such file or directory`  will be shown. After building the image the setting can be set to `true` again.
```

Comment:
Hi,

Thanks for the PR. We very appreciate that.
Do you think it is a good idea to add comprehensive documentation about the topic? For example, a draft/initial version can be found on OpenCV Q&A forum: http://answers.opencv.org/question/196039/computer-vision-annotation-tool-cvat-for-windows/.